factions/enemies_and_allies/units/hospital/hospital.xml

```diff
@@ -57,26 +57,6 @@
 			<sound enabled="false" />
 		</skill>
 
-		<skill>
```

Comment:
You can see the section I removed here weren't being used and I didn't anticipate that they would be.